library-best-practices/JLBP-17.md

```diff
@@ -0,0 +1,38 @@
+[JLBP-17] Coordinate Major Version Adoption
+-------------------------------------------
+
+It is clearly undesirable for a dependency tree to contain conflicts.
```

Comment:
I think we can delete "It is clearly undesirable for a dependency tree to contain conflicts. Unfortunately,"

If it's clear, we don't need to restate it here.